docs/paper/admin/reference/paper-plugins.md

```diff
@@ -31,32 +31,11 @@ Paper plugins are added the same as Bukkit plugins, therefore, you can follow [t
 With the introduction of Paper plugins, Paper introduces a new plugin loader that fixes some odd issues.
 However, as a result, this now causes [cyclic loading](#cyclic-plugin-loading) between plugins to no longer be supported. 
```

Comment:
This link is to its own header? I don't think it has to exist or should link to the cyclic plugin loading section in the dev docs.